runtime/src/vm/memory.rs

```diff
@@ -24,21 +24,18 @@ pub struct HandlerMemory {
   gmem: &'static Vec<u8>,
   /// Memory of the handler for fixed size data types
   pub mem: Vec<i64>,
-  /// Memory space used for closure arguments
-  closure_args: Vec<i64>,
-  fractal_closure_args: Vec<HandlerMemory>,
   /// Fractal memory storage for variable-length data types to an instance of the same struct
   pub fractal_mem: Vec<HandlerMemory>,
-  /// Helper for fractal_mem or fractal_closure_args to lookup the actual location of the relevant data, since
-  /// instantiating HandlerMemory instances for each memory usage need is expensive. -1 means the
-  /// data is in mem, otherwise it's mapping to the index in fractal_mem that houses the relevant
-  /// data.
-  either_mem: Vec<i64>,
-  pub either_closure_mem: Vec<i64>,
+  /// Helper for fractal_mem to lookup the actual location of the relevant data, since instantiating
+  /// HandlerMemory instances for each memory usage need is expensive. -1 means the data is in mem,
+  /// otherwise it's mapping to the index in fractal_mem that houses the relevant data.
+  pub either_mem: Vec<i64>,
   /// Flag indicating the memory stored is actually a pointer
   registers_ish: Vec<i8>,
```

Comment:
nit: `register_ptrs` are probably a better name now

Reply:
Yes, but I didn't want to introduce even more churn in this PR that's adding functionality. Would rather have a "cleanup" PR that intentionally changes no behavior for this.